supervision/annotators/core.py

```diff
@@ -899,6 +901,7 @@ def annotate(
             )
             center = (int(xy[detection_idx, 0]), int(xy[detection_idx, 1]))
             cv2.circle(scene, center, self.radius, color.as_bgr(), -1)
+            cv2.circle(scene, center, self.radius, (0, 0, 0), self.outer_thickness)
```

Comment:
I'd expect the background to be drawn first. I'd swap these 2 `cv2.circle` lines.